<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Secretary</class>
 <widget class="QDialog" name="Secretary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Секретарь</string>
  </property>
  <widget class="QPushButton" name="approveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="incLevelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить министра на курсы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Приказать оказать помощь министерству</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Секретарь</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
